### PR TITLE
Add ucr indicator queue

### PR DESCRIPTION
**Pull request by emord**
@gcapalbo @benrudolph

### CODE REVIEW

File: fab/environments.yml [hunk 1]
```diff
@@ -107,6 +111,8 @@ production:
         concurrency: 200
       ucr_queue:
         concurrency: 4
+      ucr_indicator_queue:
+        concurrency: 10
```
Comment (gcapalbo) [inline fab/environments.yml:116]: is 10 needed? that might be a lot

also i think it would be good to put this on celery2

Comment (gcapalbo) [reply fab/environments.yml:116]: oh nevermind, i see you're using gevent, this is fine as is

**Review by gcapalbo — COMMENTED** [review]
we should add this to staging and swiss too right?

**Comment by gcapalbo** [comment]
looks good. before releasing this it would be good to test it either locally or on staging to make sure it works as intended. i remember @benrudolph had to add some [patching](https://github.com/dimagi/commcare-hq/pull/10824) to manage.py to get celery working with gevent, so we should just make sure everything still works as intended with the patching

**Comment by emord** [comment]
Swiss doesn't have UCRs https://github.com/dimagi/commcare-hq-deploy/blob/master/fab/pillows/swiss.yml#L8

I wasn't going to add it to staging since we constantly have issues with it and didn't want to increase those problems. I will double check this works locally tomorrow morning, but ben's  changes look like it applies to all celery commands

**Comment by benrudolph** [comment]
confirming that that patching should work for all queues in the future

**Comment by gcapalbo** [comment]
yup, agreed we shouldn't have to make changes to the patching - i just meant that the patching creates an environment under which we haven't run these tasks yet so we should run them in that environment to make sure it still does what we want

**Pull request merged**